sympy/solvers/solvers.py

```diff
@@ -523,10 +524,8 @@ def solve(f, *symbols, **flags):
         * When there is a linear solution:
 
             >>> solve(x - y**2, x, y)
-            [(y**2, y)]
+            [{x: y**2}]
```

Comment:
this is inconsistent with current behavior

Reply:
Which of the two possible outputs is inconsistent?